index.html

```diff
@@ -18,17 +18,24 @@
         <div class="header">
           <div class="logo"><a href="#">My Logo</a></div>
           <nav>
+            <div class="menu-content">
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1 )template as much as possible. Please place some left, right margins on the header and use the correct color for the logo to make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160192839-5dcbea46-a42d-4236-8f42-c5f13fae0878.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160192863-483afa42-bfc5-4a28-ad2a-62dbfb7a6347.png)